src/amplitude-client.js

```diff
@@ -39,7 +39,13 @@ var AmplitudeClient = function AmplitudeClient(instanceName) {
   this._instanceName = utils.isEmptyString(instanceName) ? Constants.DEFAULT_INSTANCE : instanceName.toLowerCase();
   this._unsentEvents = [];
   this._unsentIdentifys = [];
-  this.options = { ...DEFAULT_OPTIONS, trackingOptions: { ...DEFAULT_OPTIONS.trackingOptions } };
+  this.options = {
+    ...DEFAULT_OPTIONS,
```

Comment:
Thanks for the quick fix! Could we add `ingestionMetadata` here too? (https://github.com/amplitude/Amplitude-JavaScript/blob/14c590c2eefa312bc6bce11c8baba518848ef3f0/src/options.js#L80)

Reply:
ah yeah need to add that as well

Reply:
added `ingestionMetadata` and tested

```
a.options.ingestionMetadata === b.options.ingestionMetadata
> false
```